### DISCUSSION TITLE
ErrorException Undefined array key "id" in PerformActionCalls.php

### DISCUSSION THREAD

**Discussion by Amirali-Bagheri** [livewire/livewire#3724] [7. Bugs (archived)] [answered]
Livewire Version: 2.6.1

ErrorException
Undefined array key "id" in line 27 

Illuminate\Foundation\Bootstrap\HandleExceptions::handleError
vendor/livewire/livewire/src/HydrationMiddleware/PerformActionCalls.php:27

   $id = $update['payload']['id']; have a error 

`

                if ($update['type'] !== 'callMethod') continue;

 

                $id = $update['payload']['id'];

                $method = $update['payload']['method'];

                $params = $update['payload']['params'];

 `

This is Happend After Update to 2.6.1

**Comment by joshhanley** [comment]
@Amirali-Bagheri thanks for reporting! Can you provide steps that you followed to replicate the issue, and any relevant code please?

Thanks!

**Reply by Amirali-Bagheri** [reply]
Actualy i dont know what happend exactly
I think when i click submit button this error will happend
i havnt change my code just upgrade livewire to 2.6.1

**Reply by joshhanley** [reply]
@Amirali-Bagheri no worries! Any chance you can share some of your code so we can try and replicate the issue? 🙂

**Reply by Amirali-Bagheri** [reply]
I tested again, every buttons have this error
Just very simple code

`<button type="submit" class="btn btn-primary btn-block"> Login </button>`

`class Login extends Component`
`{`
`    public function login()`
`    {`
`       dd('TEST);`
`    }`
`}`

**Reply by joshhanley** [reply]
@Amirali-Bagheri hmm thanks, I just tried that but couldn't replicate the issue. Is there any more details you can think of that I might need?

**Reply by joshhanley** [reply]
@Amirali-Bagheri do you publish your assets? If so, make sure you publish them again and then test it.

**Reply by Amirali-Bagheri** [reply]
Yes now i published again but livewire are runs fine

Screenshot
![image](https://user-images.githubusercontent.com/76995556/133341659-1f181c2d-9751-4b94-851c-70fbb45a6de7.png)

**Reply by joshhanley** [reply]
@Amirali-Bagheri ok thanks!

If you have the time, can you please share your route definition and Livewire component class, full component blade view, and any other code necessary, so I can try and replicate the issue in a test project?

Thanks!

**Reply by Amirali-Bagheri** [reply]
Suddenly I Had vary problems in my laravel. wait to fix them and rebuild composer and assets and etc... maybe solve the problems.
after that i will share the simple code here to replicate
Thanks

**Reply by joshhanley** [reply]
@Amirali-Bagheri no worries! Let me know if you manage to work it out 🙂

**Reply by brocorp-dev** [reply]
Hi guys, we encountered same issue since 2.6.x branch.

On our full page components, with or without Octane, we occurs systematically this error when we call any action :

ErrorException: Undefined array key "id"
vendor/livewire/livewire/src/HydrationMiddleware/PerformActionCalls.php:27

**Comment by joshhanley** [comment, marked as answer]
If anyone else experiences this issue and you publish Livewire's javascript assets, try publish them again first and see if that fixes the issue 🙂

You can also setup a composer script to do it automatically, have a look at the "Publishing Frontend Assets" section here for instructions https://laravel-livewire.com/docs/2.x/installation

**Reply by mlewist** [reply]
I had the same issue and publishing assets again fixed it

**Reply by Amirali-Bagheri** [reply]
issue fixed, I have some composer problems and after fix them i rebuild livewire and publish assets and finaly fixed

Thanks For Supporting. Hope The Best For Livewire :)

**Reply by joshhanley** [reply]
@Amirali-Bagheri awesome! Thanks for confirming 🙂

**Reply by DougThwaites** [reply]
Thanks @joshhanley, after publishing again all is well :)

**Reply by dgtlss** [reply]
Just came across this issue now, sadly took me some time to find this answer but so glad I did. All working now!
